src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started to work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped to work");

```

Comment:
The stopWork method in the Excavator class incorrectly references 'Bulldozer'. It should be 'Excavator stopped working'.